public/app/features/dashboard/containers/DashboardPage.test.tsx

```diff
@@ -114,138 +159,61 @@ describe('DashboardPage', () => {
   dashboardPageScenario('Dashboard init completed ', (ctx) => {
     ctx.setup(() => {
       ctx.mount();
-      ctx.setDashboardProp();
+      ctx.setDashboardProp({
+        title: 'My dashboard - Grafana',
+      });
+      ctx.rerender({ dashboard: ctx.dashboard });
     });
 
     it('Should update title', () => {
-      expect(document.title).toBe('My dashboard - Grafana');
+      expect(screen.getByText('My dashboard - Grafana')).toBeInTheDocument();
     });
 
     it('Should render dashboard grid', () => {
       expect(ctx.wrapper).toMatchSnapshot();
     });
   });
 
-  dashboardPageScenario('When user goes into panel edit', (ctx) => {
```

Comment:
These test cases should not be just removed but refactored into tests with RTL. For example, for testing edit mode we could check if the UI elements that visible only in that mode appear on the screen. Same goes for the other removed tests.

Reply:
sure. will add those 👍🏻 also

Reply:
@Clarity-89, I am not able to assert for `edit` mode tests.